scripts/process_vulnerabilities.py

```
@@ -430,13 +604,28 @@ def main():
-    vulns_to_process = filtered_vulns[:max_issues]
-    remaining_vulns = filtered_vulns[max_issues:]
-
-    if remaining_vulns:
-        print(f"⚠️  Limiting issue creation to {max_issues} vulnerabilities (out of {len(filtered_vulns)} total)")
-        print(f"   Remaining {len(remaining_vulns)} vulnerabilities will be logged but not converted to issues")
+    # Group vulnerabilities by package
+    grouped_vulns = group_vulnerabilities_by_package(filtered_vulns)
+    print(f"Grouped into {len(grouped_vulns)} packages")
+
+    # Log package summary
+    total_vuln_count = sum(pkg.get("vulnerability_count", 0) for pkg in grouped_vulns)
```

Ruff (0.14.13)

Local variable `total_vuln_count` is assigned to but never used

Remove assignment to unused variable `total_vuln_count`

(F841)